lua/autorun/cl_streamcore.lua

```diff
@@ -2,6 +2,18 @@ if SERVER then AddCSLuaFile() return end
 
 local streams = {}
 
+local streamcoreDisable = CreateClientConVar( "streamcore_disable", 0, true, false, "Disables streamcore." )
+
+hook.Add( "AddToolMenuCategories", "CFC_Streamcore_AddToolMenuCategories", function()
+    spawnmenu.AddToolCategory( "Options", "CFC", "#CFC" )
+end )
+
+hook.Add( "PopulateToolMenu", "CFC_Streamcore_PopulateToolMenu", function()
```

Comment:
We already discussed this in Discord, thank you for making your hook names fit styling guidelines :)